contract/package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "@agoric/ertp": "^0.16.3-u12.0",
+    "@agoric/time": "^0.3.3-u16.0",
```

Comment:
dapp dependencies are, unfortunately, tremendously fragile. Mixing u12 and u16 is asking for trouble, IME.

We should probably have a better scoped issue, but the tale of woe is in...
 - https://github.com/Agoric/agoric-sdk/issues/8621

Reply:
I tried rolling back to the u12 version of `@agoric/time` (60e2308) but the symptoms remain.